DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -40,7 +41,7 @@ CancellationToken cancellationToken
 
     var objectsToConvert = rootObject
       .TraverseWithPath(obj => obj is not Collection)
-      .Where(obj => obj.Item2 is not Collection);
+      .Where(obj => obj.Item2 is not Collection && obj.Item2 is not DisplayStyle && obj.Item2 is not RenderMaterial);
```

Comment:
Might it be wise to wrap these rules, they feel a but like implementation? But, I didn't get to have my traversal chat with @JR-Morgan so likely I am just lacking some context.